iosApp/Podfile.lock

```diff
@@ -1,16 +1,16 @@
 PODS:
```

Comment:
we may not even wanna commit this file, it'll fall out of date super often

Reply:
I see indeed that other hybrids don't have it version controlled either (although PHC does). In general I think it's nice to be a bit more explicit about transitive dependencies, so I'd propose to remove the file once it becomes a problem.